### PR TITLE
[BE] Extend linter to detect DOS newlines

### PR DESCRIPTION
**Pull request by malfet**
Fix DOS newlines in `onednn/decompose_silu.[cpp|h]` introduced by https://github.com/pytorch/pytorch/pull/85591 as well as one in `.github/PULL_REQUEST_TEMPLATE.md`

### CODE REVIEW

File: torch/csrc/jit/codegen/onednn/decompose_silu.cpp [hunk 1]
```diff
@@ -1,65 +1 @@
...
-void DecomposeSiluForLLGA(std::shared_ptr<Graph>& graph) {
-  DecomposeSilu(graph->block());
-  EliminateDeadCode(graph);
-}
-
-} // namespace onednn
-} // namespace fuser
-} // namespace jit
-} // namespace torch
+#include <torch/csrc/jit/codegen/onednn/decompose_silu.h>
```
Comment (izaitsevfb) [inline torch/csrc/jit/codegen/onednn/decompose_silu.cpp:1]: Are these changes related to the PR?

**Review by izaitsevfb — COMMENTED** [review]
Overall LGTM, except for the seemingly unrelated changes in
`torch/csrc/jit/codegen/onednn/decompose_silu.h`
`torch/csrc/jit/codegen/onednn/decompose_silu.cpp`

File: tools/linter/adapters/newlines_linter.py [hunk 2]
```diff
@@ -33,78 +34,86 @@ class LintMessage(NamedTuple):
...
+            return LintMessage(
+                path=filename,
+                line=idx,
+                char=None,
+                code=LINTER_CODE,
+                severity=LintSeverity.ERROR,
+                name="DOS newline",
+                original=decoded_line,
+                replacement=decoded_line[:-2] + "\n",
+                description="DOS newline found. Run `lintrunner --take NEWLINE -a` to apply changes.",
```
Comment (ZainRizvi) [inline tools/linter/adapters/newlines_linter.py:113]:
```suggestion
                description="DOS newline found. Run `lintrunner --take NEWLINE -a` to standardize newlines format.",
```

File: torch/csrc/jit/codegen/onednn/decompose_silu.cpp (hunk 1 above)
Comment (ZainRizvi) [reply torch/csrc/jit/codegen/onednn/decompose_silu.cpp:1]: +1

**Review by ZainRizvi — CHANGES_REQUESTED** [review]
cpp changes seem unrelated? Otherwise looks good

File: torch/csrc/jit/codegen/onednn/decompose_silu.cpp (hunk 1 above)
Comment (malfet) [reply torch/csrc/jit/codegen/onednn/decompose_silu.cpp:1]: Yes, it fixes the violations, see PR description

File: tools/linter/adapters/newlines_linter.py (hunk 2 above)
Comment (malfet) [reply tools/linter/adapters/newlines_linter.py:112]: Hmm, let's not make changes to description that is used throughout the file (if you want to make it, please do in a followup PR)

File: torch/csrc/jit/codegen/onednn/decompose_silu.cpp (hunk 1 above)
Comment (malfet) [reply torch/csrc/jit/codegen/onednn/decompose_silu.cpp:1]: Actually, it means that lintrunner does not work as expected, does it? Trying to fix it now

**Comment by sanchitintel** [comment]
Oops, sorry :(

**Review by huydhn — APPROVED** [review]

**Review by izaitsevfb — APPROVED** [review]
LGTM
